Makefile

```diff
@@ -0,0 +1,68 @@
+test:
```

Comment:
Lets have a test command to run all mock tests in (verbose), and another for devnet tests (also verbose)

Reply:
Mmmm..
can you tell how to run `devnet test`??